sdk/cdsclient/client_worker_v2.go

```diff
@@ -82,3 +82,12 @@ func (c *client) V2WorkerUnregister(ctx context.Context, region, runJobID string
 	}
 	return nil
 }
+
+func (c *client) V2WorkerGetSchema(ctx context.Context) error {
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+	if _, err := c.GetJSON(ctx, "/v2/jsonschema/worker-model", nil, nil); err != nil {
```

Comment:
we don't get anything out of this API call, why ?

Reply:
dead code, removed